dora/core/server/master/src/main/java/alluxio/master/scheduler/Scheduler.java

```diff
@@ -302,6 +308,8 @@ private void processJobs() {
     if (Thread.currentThread().isInterrupted()) {
       return;
     }
+    // kickstart the head task from each q of the worker if it's not running
+    mWorkerInfoHub.kickStartTasks();
```

Comment:
nit: should we process first and kickstart? although it doesn't matter when running it makes more sense when reading code

Reply:
make sense